packages/labs/ssr/src/lib/render-module.ts

```diff
@@ -32,7 +32,7 @@ export const renderModule = async (
     },
   });
   const module = await importModule(specifier, referrer, window);
-  const f = module.namespace[functionName] as Function;
+  const f = module.namespace[functionName] as (...args: unknown[]) => unknown;
```

Comment:
this means the renderModule return type is now `unknown` i guess, but the only usage i found of it casts the result anyway so should be fine?

Reply:
both `any` and `Function` trigger  a lint rule so we could just leave it as-is, with a TODO and an `eslint-disable` maybe?

Reply:
sounds good to me

Reply:
This still needs to be `any` I think, though we could do an immediate follow-up breaking change to the `3.0` branch